src/Containers/CadView.jsx

```diff
@@ -644,7 +644,7 @@ function OperationsGroupAndDrawer({deselectItems}) {
         <Box
           sx={{
             position: 'absolute',
-            top: 0,
+            top: 3,
```

Comment:
Please add a comment:

`TODO(pablo): CadView just has two sub-components the left and right group, and their first elements should be same height and offset so they line up naturally.. this is a shim for the misalignment you see with tooltips without it`